server/superdesk/io/commands/remove_expired_content.py

```diff
@@ -30,15 +30,19 @@ class RemoveExpiredContent(superdesk.Command):
     )
 
     def run(self, provider_type=None):
-        for provider in superdesk.get_resource_service('ingest_providers').get(req=None, lookup={}):
+        providers = [p for p in superdesk.get_resource_service('ingest_providers').get(req=None, lookup={})]
+        self.remove_expired({'exclude': [str(p.get('_id')) for p in providers]})
```

Comment:
Can we do like this:
self.remove_expired({'exclude': [str(p.get('_id')) for p in providers if provider.get('type') != provider_type]})

Then maybe we don't need the line 35-37